sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/_operations_mixin.py

```diff
@@ -23,6 +25,158 @@
 
 class TextAnalyticsClientOperationsMixin(object):
 
+    def analyze_status(
```

Comment:
In which scenarios will the user need to call this function?

Reply:
ohh I see so the `begin_analyze` starts the request but it is not really doing the polling or the LRO ....
The user is supposed to call this function and when status is completed then gets results. right?